src/Nonlinear/ReverseAD/forward_over_reverse.jl

```diff
@@ -328,10 +332,38 @@ function _forward_eval_ϵ(
                         recip_denominator,
                     )
                 elseif op > 6
-                    error(
-                        "User-defined operators not supported for hessian " *
-                        "computations",
+                    f_input = _UnsafeVectorView(d.jac_storage, n_children)
+                    for (i, c) in enumerate(children_idx)
+                        f_input[i] = ex.forward_storage[children_arr[c]]
+                    end
+                    H = _UnsafeLowerTriangularMatrixView(
+                        d.user_output_buffer,
```

Comment:
Where is it enforced that user_output_buffer is large enough?

Reply:
It calls this contractor: https://github.com/jump-dev/MathOptInterface.jl/pull/1819/files#diff-b6954ec7398499723bd82286762989c12564fda78890c07d49a3c1c06e0ea3a7R116

Reply:
So the constructor could break other unsafe views on the same array. That sounds extra unsafe... At least needs to be documented.

Reply:
Yes. For now though, the only usage of this is evaluating the user-defined hessians, and we use the `.user_output_buffer` as storage, so it's a safe usage of the unsafe behavior.